server/database/config/build.sql

```diff
@@ -0,0 +1,74 @@
+BEGIN;
+
+DROP TABLE IF EXISTS "user",parent,teacher,student,teacher_student,"subject",subject_teacher,activity,homework CASCADE;
```

Comment:
add line after this line 
add space after the colon 
```suggestion
DROP TABLE IF EXISTS "user", parent, teacher, student, teacher_student, "subject", subject_teacher, activity, homework CASCADE;

```